pkgs/development/dotnet-modules/python-language-server/default.nix

```diff
@@ -59,11 +59,11 @@ stdenv.mkDerivation {
 
   dontStrip = true;
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "Microsoft Language Server for Python";
     homepage = "https://github.com/microsoft/python-language-server";
-    license = stdenv.lib.licenses.asl20;
-    maintainers = with stdenv.lib.maintainers; [ thomasjm ];
-    platforms = with stdenv.lib.platforms; linux;
+    license = licenses.asl20;
+    maintainers = with maintainers; [ thomasjm ];
+    platforms = platforms.linux;
```

Comment:
Just curious, is there a reason why this is restricted to Linux only? Would it not work on aarch64 (ARM) or darwin (Mac)?

Reply:
I have no idea...googling around it seems like the dotnet CLI tools can be installed on those platforms. If I add more platforms here and push will the CI try it out?

Reply:
Well, I added Darwin and the CI didn't seem to mind. It doesn't look like most developer tools support aarch64 so I'm going to leave that out for now, let me know if you disagree.

Reply:
Oh and Linux and Darwin are the platforms that VS Code supports FWIW

Reply:
the ``aarch64`` (i.e. ARM) build fails on OfBorg. Can you change this to ``platforms.x86_64-linux`` or ``platforms.x86_64`` (think the first one works, not sure about second, see full list at https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix
So please either fix the aarch64 build (see [OfBorg log](https://logs.nix.ci/?key=nixos/nixpkgs.85930&attempt_id=22fae708-acd9-454f-91a8-fbe9622945ae)) or block aarch64 building.

Reply:
It's either `platforms = [ "x86_64-linux" ];` or `platforms = platforms.x86_64;`, where the latter consists of:

```
[ "x86_64-cygwin" "x86_64-freebsd" "x86_64-linux" "x86_64-netbsd" "x86_64-openbsd" "x86_64-solaris" "x86_64-darwin" "x86_64-windows" "x86_64-none" "x86_64-genode" ]
```

Reply:
Okay, going with `platforms = [ "x86_64-linux" ];`